### ISSUE TITLE
List file on a non-existent repo/branch should throw an error instead of failing silently

### ISSUE THREAD

**Issue by gabrielgrant** [pachyderm/pachyderm#3083]
**What happened?**:

Fat fingered  a `list-file` command and, instead of seeing an error, it just seemed as though the repo was empty:

```
$ pachctl list-file attribues master  # note missing "t"
$ pachctl list-file attributes maser  # note other missing "t"
$ pachctl list-file attributes master
NAME  TYPE SIZE 
1.csv file 32B  
2.csv file 192B 
```

**What you expected to happen?**:

An error message telling me the repo/branch doesn't exist

**How to reproduce it (as minimally and precisely as possible)?**:

```
pachctl create-repo test
echo hello | pachctl put-file test master hello
pachctl list-file test maser
pachctl list-file tes maser
pachctl list-file test master
```

**Anything else we need to know?**:

glob-file seems to error as expected (though the error message is kinda confusing and ugly):

```
$ pachctl glob-file test maser '*'
pachyderm/1.7.0/pachyderm_pfs/branches/test/maser not found
```

**Environment?**:
- Kubernetes version (use `kubectl version`):
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.1", GitCommit:"b1b29978270dc22fecc592ac55d903350454310a", GitTreeState:"clean", BuildDate:"2018-07-17T18:53:20Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```

- Pachyderm CLI version (use `pachctl version`):
```
$ pachctl version
COMPONENT           VERSION             
pachctl             1.7.4               
pachd               1.7.4             
```
- Cloud provider (e.g. aws, azure, gke) or local deployment (e.g. minikube vs dockerized k8s): minikube

**Comment by mdaniel** [comment]
I don't know if adding the :+1: reaction registers with the issue owner, but I would also like this fixed

**Comment by Tryneus** [comment]
This has been fixed by #3596 and merged into `master` - it should be in `v1.8.7` and `v1.9.0`.
